test/core/data.wast

```diff
@@ -158,28 +158,28 @@
 
 ;; Invalid bounds for data
 
-(assert_unlinkable
+(assert_trap
```

Comment:
Why do you change the error type here to `assert_trap`?

Reply:
Because we decided that active segments should be equivalent to invoking `memory.init` now, so the linking semantics is slightly more relaxed and this case becomes a regular trap.